Cleaned up vote method.

Dr-Horv opened the pull request.

Dr-Horv committed:
Cleaned up vote method.

lindskogen commented on app.js:
```diff
@@ -143,42 +143,60 @@ function countDown() {
-        if (codes[i][vote.id - 1] == req.body.code) {
-            validCode = true;
-            break;
+        if (codes[i][vote.id - 1] == code) {
```
Tip, use: `Array.any`

Dr-Horv committed:
More code cleaning

Dr-Horv committed:
Further cleaning

lindskogen committed:
further refactoring

Dr-Horv committed:
Corrected variable names and syntax errors

Dr-Horv committed:
Fixed initial state not being noVote

Dr-Horv committed:
Working on vote functionaility

Dr-Horv committed:
Finished countVotes, still untested

Dr-Horv committed:
Fixed syntax errors

lindskogen committed:
bugfixes

krabban91 committed:
Vote page partially fixed the ux.

lindskogen committed:
we have written a test

lindskogen committed:
testsssssss

lindskogen committed:
app.locals should not be in view

lindskogen committed:
Extract voteCounter and create voteCounter-test
Add missing semicolons

lindskogen committed:
Change map to forEach

lindskogen committed:
Semicolons!

lindskogen committed:
Refactor most of app to use a simple list of codes for validation
Tests for voteManager

lindskogen committed:
CodeManager and tests for codeManager

lindskogen committed:
test coverage for utils

lindskogen committed:
add build status flag

lindskogen committed:
small styles

lindskogen committed:
Travis configuration file

lindskogen committed:
html style cleanup

lindskogen committed:
Actual design for voteIT.

Separate backend and frontend - made into a React application.
Application calls /status and gets initial information.
Then builds fields for every candidate in the vote.

lindskogen committed:
Codelength variable missing

lindskogen committed:
Reformulate label in admin interface

lindskogen committed:
Random generator swapped to browserCrypto.
Expose randomToken function

lindskogen committed:
Complete refactoring to react views
Open printer page in new window

lindskogen committed:
Admin able to stop session - Present winners
Add confirm buttons for recreate codes and stop session
Remove timeLeft
User gets status every minute
Admin sign out if 401

lindskogen committed:
General integration tests, UX fixes

lindskogen committed:
present error messages to user

lindskogen committed:
Prettify await screen
